tests/sites/version_switcher/page2.rst

```diff
@@ -0,0 +1,9 @@
+:html_theme.sidebar_secondary.remove: true
```

Comment:
ditto the prior [nitpick] comment; a site that only tests the version switcher probably only needs a single page, and if it needs multiple pages, they shouldn't be doing unrelated things like hiding sidebars.